libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -795,8 +795,10 @@ void AP_GPS::update(void)
 
 #if defined(GPS_BLENDED_INSTANCE)
     // copy the primary instance to the blended instance in case it is enabled later
-    state[GPS_BLENDED_INSTANCE] = state[primary_instance];
-    _blended_antenna_offset = _antenna_offset[primary_instance];
+    if (primary_instance != GPS_BLENDED_INSTANCE) {
```

Comment:
JUST NUKE THIS ALREADY! Sorry to be a bear on it but I've pointed this out in review before: https://github.com/ArduPilot/ardupilot/pull/12844#discussion_r347533919 It's dead code that doesn't make sense. If we had removed it when first brought up we never would have had the problem...

Reply:
I may have pushed for the minimum change.  If we're sure it can just be nuked then I'm OK with that and I'm happy to re-test afterwards..

Reply:
done